structuralcodes/materials/constitutive_laws.py

```diff
@@ -302,6 +302,87 @@ def get_ultimate_strain(
         return (100, self._eps_u)
 
 
+class Sargin(ConstitutiveLaw):
```

Comment:
Just wondering, since it is meant to be used for concrete, should it be called `SarginConcrete`?

Reply:
mmm.. I am not against it, but in general I find the names should be kept as the names of the model, otherwise we risk having SarginConcrete, KentParkConcrete, ThoeredolfConcrete etc...
Do you agree?

Reply:
Let us leave it like it is.